### DISCUSSION TITLE
upload and multer not working with koajs and knex driver

### DISCUSSION THREAD

**Discussion by f-i-t-s-u-m** [feathersjs/feathers#2958] [Q&A]
Two years ago, I was using Feathersjs v4 and successfully implemented file uploads using Feather-blobs and Multer. Now that Feathersjs has upgraded to version 5, however, I am unsure of how to go about implementing file uploads on services for a new project.
I also try to follow https://feathersjs.com/cookbook/express/file-uploading.html#feathers-blob-with-multipart-support and try to use it on koajs still no luck.

**Comment by arcanys-ivan** [comment]
any updates for this one? seems like we have same issue

**Reply by 8BallBomBom** [reply]
For future reference.
Related response over [here](https://github.com/feathersjs/feathers/discussions/3097).
